crates/sdk/src/network/mod.rs

```diff
@@ -24,3 +24,5 @@ pub use error::*;
 pub(crate) const DEFAULT_NETWORK_RPC_URL: &str = "https://rpc.production.succinct.xyz/";
 pub(crate) const DEFAULT_TIMEOUT_SECS: u64 = 14400;
 pub(crate) const DEFAULT_CYCLE_LIMIT: u64 = 100_000_000;
+pub(crate) const DEFAULT_VM_MEMORY_KB: u64 = 100_000;
+pub(crate) const MEMORY_PER_ADDRESS: u64 = 4;
```

Comment:
nit: this should be BYTES_PER_WORD

Reply:
Fixed: https://github.com/succinctlabs/sp1/pull/2092/commits/68335c9fb04e0100ae9a183b0c8d8fc6adcada21